src/main/java/org/apache/commons/compress/archivers/sevenz/SevenZFile.java

```diff
@@ -1361,7 +1361,7 @@ private void sanityCheckFilesInfo(final ByteBuffer header, final ArchiveStatisti
                 }
             }
         }
-        stats.numberOfEntriesWithStream = stats.numberOfEntries - (emptyStreams > 0 ? emptyStreams : 0);
+        stats.numberOfEntriesWithStream = stats.numberOfEntries - (Math.max(emptyStreams, 0));
```

Comment:
Parens around the call to max are not needed.

Reply:
done.